tssh/agent_unix.go

```diff
@@ -0,0 +1,39 @@
+//go:build !windows
+
+package tssh
+
+import (
+	"net"
+	"os"
+
+	"golang.org/x/crypto/ssh/agent"
+)
+
+func getAgentSigners() []*sshSigner {
+	socket := os.Getenv("SSH_AUTH_SOCK")
+	if socket == "" {
+		return nil
+	}
+
```

Comment:
Should we use `DialTimeout`?

Reply:
Yeah, that would be better, I'll make a commit for it.